lib/flutter_camera_ml_vision.dart

```diff
@@ -335,11 +329,8 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>>
   }
 
   Widget _getPicture() {
-    if (_lastImage != null) {
-      final file = File(_lastImage);
-      if (file.existsSync()) {
-        return Image.file(file);
-      }
+    if (_lastFrameFile != null && _lastFrameBytes != null) {
```

Comment:
so you only need the bytes here, so `_lastFrameFile ` is not needed at all I would not keep a reference to it in that case